microhab/SPEXpdd.tex

```diff
@@ -0,0 +1,254 @@
+%% *************************************************************************
```

Comment:
I would recommend renaming the document iteslf to "micro-hab_PDD.tx" or something along those lines

Reply:
+1 to renaming the `.tex` file to something else